src/main/java/com/shapesecurity/bandolier/es2017/bundlers/BundlerOptions.java

```diff
@@ -0,0 +1,59 @@
+package com.shapesecurity.bandolier.es2017.bundlers;
+
+import javax.annotation.Nonnull;
+
+public final class BundlerOptions {
+
+	public enum ImportUnresolvedResolutionStrategy {
+		COMPILE_ERROR,
+		DEFAULT_TO_UNDEFINED,
+		THROW_ON_REFERENCE,
+		NOTHING
+	}
+
+	public enum ExportStrategy {
+		EXPLICIT,
+		ALL_GLOBALS
+	}
+
+	public enum DangerLevel {
+		SAFE, // to spec, should be used unless one wants compile errors when optimizations are impossible (or depends on Module[Symbol.toStringTag] behavior)
+		BALANCED, // removes TDZ handling, Symbol.toStringTag, throwOnCircularDependency provides greedy safety for TDZ removal, however, Symbol.toStringTag remains unsafe.
+		DANGEROUS // removes read-only protection for imports, does nothing over BALANCED with throwOnImportAssignment enabled.
+	}
+
+	@Nonnull
+	public final ImportUnresolvedResolutionStrategy importUnresolvedResolutionStrategy;
+	@Nonnull
+	public final ExportStrategy exportStrategy;
+	@Nonnull
+	public final DangerLevel dangerLevel;
+	public final boolean throwOnCircularDependency;
+	public final boolean throwOnImportAssignment;
+
+
+	private BundlerOptions(@Nonnull ImportUnresolvedResolutionStrategy importUnresolvedResolutionStrategy, @Nonnull ExportStrategy exportStrategy, @Nonnull DangerLevel dangerLevel, boolean throwOnCircularDependency, boolean throwOnImportAssignment) {
```

Comment:
I think we should make this public. It'd be annoying to have to start from one of `NODE_OPTIONS` or `SPEC_OPTIONS`.